tests/unit/test_handlers.py

```diff
@@ -1073,6 +1073,29 @@ def test_set_operation_specific_signer_s3v4_unsigned_payload(self):
         self.assertEqual(response, 's3v4')
         self.assertEqual(context.get('payload_signing_enabled'), False)
 
+    def test_set_operation_specific_signer_defaults_to_asterisk(self):
+        signing_name = 'myservice'
+        context = {
+            'auth_type': 'v4a',
+        }
+        handlers.set_operation_specific_signer(
+            context=context, signing_name=signing_name
+        )
+        self.assertEqual(context['signing']['region'], '*')
+
+    def test_set_operation_specific_signer_prefers_client_config(self):
+        signing_name = 'myservice'
+        context = {
```

Comment:
We should also have a region declared in a signing context here to demonstrate it won't be used if this config is present.